sass/elements/_elements.scss

```diff
@@ -2,3 +2,4 @@
 @import "banner";
 @import "index-schedule";
 @import "navbar";
+@import "schedule";
```

Comment:
I agree with this change---having the sass for this in `main` makes no sense (I have a feeling I did it as a stopgap).